.golangci.yml

```diff
@@ -3,22 +3,23 @@ run:
   go: "1.22"
   timeout: 3m
   allow-parallel-runners: true
+
 output:
   formats: colored-line-number
 
 linters:
   disable-all: true
   enable:
-    - gosimple  # Suggest code simplifications
-    - govet     # Reports suspicious constructs
-    - ineffassign  # Detects unused variable assignments
-    - staticcheck  # go vet on steroids
-    - unused    # Detect unused constants, variables, functions, and types
-    - gocyclo   # Detect cyclomatic complexity
-    - gofmt # Enforce standard formatting
+    - gosimple # Suggest code simplifications
+    - govet # Reports suspicious constructs
+    - ineffassign # Detects unused variable assignments
+    - staticcheck # go vet on steroids
+    - unused # Detect unused constants, variables, functions, and types
+    - gocyclo # Detect cyclomatic complexity
+    - gofmt
```

Comment:
sorry, I guess I change back from gofumpt to gofmt. Any reason for using gofumpt, becoz I remember using gofumpt giving me some linter error, which I can't able to solve at that time.